lib/inertia_rails/lazy.rb

```diff
@@ -0,0 +1,23 @@
+class InertiaRails::Lazy
+  def initialize(value = nil, &block)
+    @value = value
+    @block = block
+  end
+
+  def call
+    to_proc.call
+  end
+
+  def to_proc
```

Comment:
there might be a more elegant way to express this, but it does get the tests green!